modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -66,11 +70,11 @@ spec:
 <1> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
 <2> The secondary workload, which is not compliant with the PodSpec.
 <3> The custom location of the container path.
-
++
```

Comment:
Are you sure you want to align the following lines and example with the preceding callout item #3?